src/applications/vaos/components/AppointmentTime.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatInTimeZone } from 'date-fns-tz';
+import { stripDST } from '../utils/timezone';
+
+/**
+ * Component for displaying appointment time in a consistent format with timezone
+ */
+export default function AppointmentTime({ date, timezone }) {
+  const formattedTime = stripDST(
+    formatInTimeZone(new Date(date), timezone, 'h:mm aaaa'),
+  );
+  const timezoneAbbreviation = stripDST(
+    formatInTimeZone(new Date(date), timezone, 'zzz'),
+  );
+  const timezoneDescription = stripDST(
+    formatInTimeZone(new Date(date), timezone, 'zzzz'),
```

Comment:
Hmm I'm surprised by this behavior for the unit tests since the documentation says it'll just return the GMT timezones with a leading 0: https://date-fns.org/docs/format. Maybe that only occurs for the Pacific/* and Asia/* timezones? Can you try this for Samoa Time, Chamorro time or Philippine time to see what happens?

Reply:
This is the date-fns-tz formatInTimeZone function so it behaves differently. https://www.npmjs.com/package/date-fns-tz#date-and-time-zone-formatting

Reply:
Ah interesting, this is good to know. I'm curious for these types of descriptions whether we should keep the `Standard/Daylight` portions in the descriptions given that we are stripping those out from the abbreviations. I would check with the UAE UX team for that but I'll leave it up to you if you want to do that as part of this PR or if you'd like to do that as a separate followup.